doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,672 @@
+   
+       // Update the display when the element is loaded in the DOM
+       Loaded += (snd, evt) => UpdateDisplay(newLanguage: Language);
+   }
+   ```
+
+5. Compile & run.  It should work like this:
+   ![image-20200415135422628](image-20200415135422628.png)
+
+## 🔬 Going further
+
+That's a very simple integration because there's no _callback_ from HTML to managed code and _PrismJS_ is a self-contained framework.
+
+Few improvements could be done to make the code more _production ready_:
+
+* **Make the control multi-platform**. The simple way would be to use a WebView on other platforms: that way the exact same text-rendering framework would be used everywhere. Right now this code won't compile on other targets.
+* **Create script files instead of generating dynamic javascript**. That would have the advantage of improving performance and increase the ability to debug it. Few projects are also using TypeScript to generate javascript. This approach is done by Uno itself for the `Uno.UI.Wasm` project: https://github.com/unoplatform/uno/tree/master/src/Uno.UI.Wasm.
+* **Support more PrismJS features**. There's many [_plugins_ to PrismJS](https://prismjs.com/#plugins) you can play with. Most of them are very easy to implement.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+# Article 2 - Callback to app from JavaScript
+
+In the previous article, a simple _syntax highlighter_ were used to enhance the display of text in HTML. But it's not enough for most apps: it's often required for JavaScript components to _call back_ to the application. The easiest way to do that in Uno-Wasm is by using [_DOM Events_](https://developer.mozilla.org/docs/Web/Guide/Events/Creating_and_triggering_events). Let's create an application illustrating how to use them. Application using Uno can [consume DOM Events and CustomEvents](https://platform.uno/docs/articles/wasm-custom-events.html) very easily.
+
+# Sample 2 - Integration of Flatpickr
+
+📝 [Flatpickr](https://flatpickr.js.org/) is a lightweight, self-contained _Date & Time Picker_. It's an easy way to explore how a JavaScript can call-back to _managed application code_. In this case, it will be used to report when the picker is opened and the picked date & time.
+
+## 1. Create the solution in VisualStudio
+
+📝 This part is very short because it's similar to previous article:
+
+1. Create a `Cross-Platform App (Uno Platform)` project and name it `FlatpickrDemo`.
+2. Remove `.Droid`, `.iOS` & `.UWP` projects from solution.
+3. Update to latest _stable_ version of `Uno.*` dependencies.
+
+## 2. Inject Flatpickr from CDN
+
+🎯 This section is using a CDN to get _Flatpickr_ instead of hosting the javascript directly in the application. It's not always the best solution - it creates a dependency on the Internet availability and any change made server-side could break the application. But the goal here is to demonstrate the flexibility of Uno, not how to build the best application architecture.
```

Comment:
```suggestion
🎯 This section is using a CDN to get _Flatpickr_ instead of hosting the javascript directly in the application. It is not always the best solution as it creates a dependency on the Internet availability. Any change made server-side could break the application.
```

Reply:
I agree...